lib/ignore.js

```diff
@@ -28,6 +28,10 @@ function ignore(dir, options) {
       // add the xpi file name to the ignore list
       var xpiName = getID(manifest) + "*.xpi";
       defaultRules.push(xpiName);
+      // add the signed xpi file name to the ignore list
+      var signedXpiName = manifest.title.toLowerCase().replace(/ /g, "_") + "*.xpi";
+      defaultRules.push(signedXpiName);
+      console.log(signedXpiName);
```

Comment:
This console statement isn't necessary. You should be able to see info about the rules when using `--verbose`.